README.md

```diff
@@ -13,4 +13,6 @@ import 'core-js/modules/web.dom-collections.for-each';
 
 You will need:
 
-`npm i core-js@3.1.3`
\ No newline at end of file
+`npm i core-js@3.1.3`
+
+You will also need an appropriate `Promise` and `fetch()` polyfill.
```

Comment:
As we're recommending core-js can we just add the relevant polyfills to the code block above?

Reply:
We can suggest https://github.com/UniversityofWarwick/scholarships/blob/3f30d418219130b5c8f797423988780b31c491b3/app/assets/js/polyfills.js#L3 et al (DOM collection etc) but we'll still need a `fetch` polyfill

Perhaps we recommend https://github.com/github/fetch directly?

Reply:
Fine with me